src/components/sprint-view-table/tasks-table.tsx

```diff
@@ -0,0 +1,190 @@
+          return 0;
+        }
+      }));
+      setTimeEntries(fetchedTimeEntries);
+    } catch (error) {
+      console.error(`Error fetching time entries: ${error}`);
+    }
+  };
+
+  /**
+   * Retrieve total time entries for a task
+   */
+  const getTotalTimeEntries = (task: Tasks) => {
+    if (timeEntries.length!==0) {
+      return timeEntries[tasks.indexOf(task)];
+    }
+    return 0;
+  }
+
+  /**
+   * Lowercase all letters in a string except the first letter except first one
+   */
+
+  const lowerCaseAllWordsExceptFirstLetters = (string: string) => {
+    return string.replace(/\S*/g, (word) => {
+      return word.slice(0, 1) + word.slice(1).toLowerCase();
+    });
+  };
+
+  return ( 
+    <Card key={0} sx={{ backgroundColor: '#f2f2f2', margin: 0, paddingTop: "5px", paddingBottom: "5px", width: "100%", height: "100", marginBottom: "16px", 
+    '& .high_priority': {
+      color: 'red'}, 
+    '& .low_priority': {
+      color: 'green'} 
+    }}>
+      <IconButton onClick={() => setOpen(!open)}>
+        {open ? <KeyboardArrowUpIcon /> : <KeyboardArrowDownIcon />}
+      </IconButton>
+      <span>{project ? project.name : ""}</span>
+      {open && 
+        <>
+          {loading ? 
+            <Box sx={{textAlign: 'center'}} >
+              <CircularProgress sx={{ scale: "100%", mt: "3%", mb: "3%"}} />
+            </Box>
+            :
+            <DataGrid
+              sx={{
+                backgroundColor: "#f6f6f6",
+                borderTop: 0,
+                borderLeft: 0,
+                borderRight: 0,
+                borderBottom: 0,
+                '& .header-color': {
+                  backgroundColor: '#f2f2f2',
+                }
+              }}
+              autoHeight={true}
+              localeText={{ noResultsOverlayLabel: strings.sprint.notFound }}
+              disableColumnFilter
+              hideFooter={true}
+              filterModel={
+                {
+                items: [{ 
+                  field: 'status',
+                  operator: 'contains', 
+                  value: filter
+                }]
+              }}
+              rows={tasks}
+              columns={[
+                { 
+                  field: 'title',
+                  headerClassName: 'header-color', 
+                  headerName: strings.sprint.taskName,
+                  minWidth: 0,
+                  flex: 3
+                },
+                { 
+                  field: 'assignedPersons',
+                  headerClassName: 'header-color',         
+                  headerName: strings.sprint.assigned, 
+                  flex: 1
+                },
+                { 
+                  field: 'status',
+                  headerClassName: 'header-color', 
+                  headerName: strings.sprint.taskStatus, 
+                  flex: 1, valueGetter: (params) => lowerCaseAllWordsExceptFirstLetters(params.row.statusCategory || ''),
```

Comment:
If the statusCategory does not exist on the task then this should be removed for now.

Reply:
"status" will be used instead of "statusCategory"